dev/EnvironmentManager/ChangeTrackerHelper/ChangeTrackerHelper/ChangeTrackerHelper.h

```diff
@@ -0,0 +1,56 @@
+﻿#pragma once
+#include <windows.h>
+#include <appmodel.h>
+
+enum AreChangesTracked
+{
+    YES,
+    NO,
+    NOT_CHECKED
+};
+
+AreChangesTracked m_AreChangesTracked{NOT_CHECKED};
```

Comment:
`m_...`?

This is a global variable? If so then it should be `g_...`

Reply:
This file is removed.